Features/Context/SubContext/Authentication.php

```diff
@@ -51,6 +51,7 @@ public function goToPlatformUiAndLogIn($username, $password)
 
     /**
      * @Given I am logged in as admin on PlatformUI
+     * @Given I am logged in as an Editor in PlatformUI
      */
     public function loggedAsAdminPlatformUi()
```

Comment:
I agree, could one of you do a separate PR on this once we have travis up and running for PR sanity?